apps/service_persist/test/persist_test.exs

```diff
@@ -124,9 +113,8 @@ defmodule PersistTest do
         id: "persist-1",
         dataset_id: "ds1",
         subset_id: "example",
-        source: "topic-example",
-        destination: "ds1_example",
-        schema: []
+        source: Source.Fake.new(),
+        destination: "ds1_example"
```

Comment:
Don't both of these need to be the new type of protocol struct?

Reply:
Same deal here. Not yet.